src/ctree-app/ctree-landing.html

```diff
@@ -392,10 +388,6 @@
         background-color: #F8F8F8;
         color: #196848;
       }
-      #features.section-styling {
-        background-color: #FFFFFF;
-        color: #196848;
-      }
```

Comment:
Nice, I'm glad to see you cleaned up all of these!  A good thing to take note of as a best practice for anyone else checking out this PR.

Reply:
You were absolutely right. Thanks for the insight.